src/platforms/javascript/guides/ember/configuration/other-versions/ember2.mdx

```diff
@@ -1,41 +0,0 @@
----
-title: Ember 2.x
```

Comment:
Yeah, I removed these files because these things actually don't even exist anymore - we do not have this integration anymore 😅 so Instead of "fixing" the CDN docs there, I just removed them!